src/Tizen.Wearable.CircularUI.Chart.Forms/LineDataItemGroup.cs

```diff
@@ -0,0 +1,83 @@
+﻿/*
+ * Copyright (c) 2018 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Flora License, Version 1.1 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://floralicense.org/license/
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System.Collections.Generic;
+using Xamarin.Forms;
+
+namespace Tizen.Wearable.CircularUI.Chart.Forms
+{
+    /// <summary>
+    /// LineDataItemGroup is an inheritance type of DataItemGroup for the LineChartView.
+    /// </summary>
+    public class LineDataItemGroup : DataItemGroup
+    {
+        Color _fgColor;
+        byte _fgColorAlpha;
```

Comment:
You can remove all constructor of `LineDataItemGroup` if change like below
```suggestion
        Color _fgColor = Color.Transparent;
        byte _fgColorAlpha = 0xFF;
```

Reply:
Done. but remain constructor preventing build error when user make new LineDataItemGroup with parameters.

Reply:
Could you explain more details?
I don't understand which error was occurred?

Reply:
After I removed all  constructor in `LineDataItemsGroup`, I got the following error.
"LineDataItemGroup has no constructor using 2 parameter "

```c#
var lineDataItemGroup = new LineDataItemGroup(colorDummyDataItems, "Weekly data with each color and forground color");
```

Reply:
hm.. base class has a constructor with two parameter, so i think, something wrong..

Reply:
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-messages/cs1729

LineDataItemGroup class has only a parameterless constructor. it make cs1729 error when build the following code.

```c#
var lineDataItemGroup = new LineDataItemGroup(colorDummyDataItems, "Weekly data with each color and forground color");
```

Reply:
Yes, you are right, my mistake